schema.sql

```diff
@@ -16,6 +17,7 @@ CREATE TABLE IF NOT EXISTS `players` (
   `name` varchar(255) NOT NULL,
   `group_id` int NOT NULL DEFAULT '1',
   `account_id` int NOT NULL DEFAULT '0',
+  `main_character` tinyint NOT NULL DEFAULT '0',
```

Comment:
You also need to update https://github.com/otland/forgottenserver/blob/master/schema.sql#L382